karma.conf.js

```diff
@@ -0,0 +1,36 @@
+const wpConf = require("./webpack.config")
+wpConf.devtool = "inline-source-map"
+
+module.exports = config => {
+    let findTests = () => {
+        // THIS is why I hate NodeJS. That tool is so immature that you always
+        // have to install a bunch of crappy modules to do some tiny operation
+        // Long life to Python \o/
+        let glob = require("glob")
+        return glob.sync("./socialhome/**/app/tests/**/*\.tests\.js")
```

Comment:
I probably should have better read [the documentation](https://karma-runner.github.io/0.13/config/configuration-file.html):

>All of the configuration options, which specify file paths, use the minimatch library to facilitate flexible but concise file expressions so you can easily list all of the files you want to include and exclude.

This function seems unneeded.